jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -114,42 +114,95 @@ public static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	private String imagesDir = null;
+	private boolean leaveRunning = false;
+	private boolean shellJob = true;
+	private boolean extUnixSupport = true;
+	private int logLevel = 0;
+	private String logFile = null;
+	private boolean fileLocks = false;
+	private String workDir = null;
```

Comment:
I'd prefer to see fields initialized only in the constructor, and then only those fields were the value is not the default.